INSTALL.md

```diff
@@ -9,30 +9,28 @@ To run the commands in the README.md the following items are required
 1. Python development, install with `sudo apt install -y python3-dev`
 1. [Emscripten SDK](https://emscripten.org/docs/getting_started/downloads.html)
 1. [Docker Engine](https://docs.docker.com/install/), setup so `docker` command can be run [without sudo](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
+1. [Perl](https://www.perl.org/), already installed on Linux
 
-## Generating code from Markdown
-
-Entangled is used to convert code blocks in Markdown to source code files.
+## Command collection
 
-1. Install [entangled](https://github.com/entangled/entangled)
-2. Run entangled daemon with
+All the commands in the [README.md](README.md) and [CONTRIBUTING.md](CONTRIBUTING.md) can be captured in a [Makefile](https://en.wikipedia.org/wiki/Makefile) like so:
 
-```shell
-entangled README.md INSTALL.md
-```
+```{.makefile file=Makefile}
+# this Makefile snippet is stored as Makefile
+.PHONY: clean clean-compiled clean-entangled test all check entangle entangle-list py-deps start-redis stop-redis run-webservice run-celery-webapp run-webapp build-wasm host-files test-wasm
 
-Or the [Entangled - Pandoc filters](https://github.com/entangled/filters) Docker image can be used
+UID := $(shell id -u)
+# Prevent suicide by excluding Makefile
+ENTANGLED := $(shell perl -ne 'print $$1,"\n" if /^```\{.*file=(.*)\}/' *.md | grep -v Makefile | sort -u)
+COMPILED := bin/newtonraphson.exe src/py/newtonraphsonpy.*.so apache2/cgi-bin/newtonraphson src/js/newtonraphsonwasm.js  src/js/newtonraphsonwasm.wasm
```

Comment:
It will be easier if you could use file extesions here. 

'.*\.(exe||so|js|wasm)$'

Reply:
Let's KISS.
I think being explicit here is better, because the paths are used in the rest of the Makefile.

Reply:
:kissing_cat: